mobile/build.gradle

```diff
@@ -64,7 +64,7 @@ dependencies {
     implementation "com.google.android.gms:play-services-vision:$playServicesVersion"
     implementation "com.google.firebase:firebase-config:$playServicesVersion"
     implementation 'com.j256.ormlite:ormlite-android:5.1'
-    implementation 'com.mikepenz:materialdrawer:6.0.7'
+    implementation 'co.zsmb:materialdrawer-kt:1.3.4'
```

Comment:
Why is this change needed? Also I think the Java API is pretty reasonable and I don't feel like having a third-party wrapper. I prefer to reverting this change.